src/dotnet/ToolManifest/ToolManifestFinder.cs

```diff
@@ -20,6 +21,8 @@ internal class ToolManifestFinder : IToolManifestFinder
         private readonly IFileSystem _fileSystem;
         private const string _manifestFilenameConvention = "localtool.manifest.json";
 
+        // The supported tool manifest file version.
+        private const int SupportedVersion = 1;
```

Comment:
NIT: empty line between the const and the constructor declaration?

Reply:
fixed